src/doppler-types.ts

```diff
@@ -13,7 +13,13 @@ export type EmptyResultWithoutExpectedErrors = { success: true } | UnexpectedErr
 
 export type PathType = 'free' | 'standard' | 'plus' | 'agencies';
 
-export type PlanType = 'free' | 'prepaid' | 'monthly-deliveries' | 'subscribers';
+export type PlanType =
+  | 'free'
+  | 'prepaid'
+  | 'monthly-deliveries'
+  | 'subscribers'
+  | 'agencies'
+  | 'demo';
```

Comment:
Es `demo` un tipo de plan? como entra en los paths existentes?

Reply:
Demo is a type that overlaps with free, it's a free with some other functionalities enabled. Demo is supposed to behave as free in paths.

Reply:
No estoy seguro de que sea buena idea meterlo como un tipo, nos hace el modelo más complejo.

Por ejemplo, tenés tipos para los planes free, pero me parece que no tenés para el demo.

Reply:
I will think a way of doing without this demo type, seems about right.